dev/com.ibm.ws.opentracing/src/com/ibm/ws/opentracing/OpentracingContainerFilter.java

```diff
@@ -40,12 +43,15 @@
 import io.opentracing.propagation.TextMap;
 import io.opentracing.tag.Tags;
 
+import org.osgi.service.component.annotations.Component;
+
 /**
  * <p>Container filter implementation.</p>
  *
  * <p>This implementation is stateless. A single container filter is used by all applications.</p> *
  */
-public class OpentracingContainerFilter implements ContainerRequestFilter, ContainerResponseFilter, ExceptionMapper<Throwable> {
+@Component
```

Comment:
this class will need to be added to the bnd file under `-dsannotations`

Also, it should have the same annotation attributes that I mentioned in the 1.1 project.